routes/xref/index.js

```diff
@@ -2,7 +2,24 @@
 const { search } = require("respec-xref-route/search");
 
 module.exports.route = function route(req, res) {
-  const { keys, options } = req.body;
+  const { keys = [], options } = req.body;
   const body = search(keys, options);
+
+  const errors = getErrorCount(body.result);
+  if (errors / keys.length > 0.05) {
+    // add to logs if error rate is more than 5%
```

Comment:
Should we send all to logs instead or 5% rate is good? (or `if (errors.length)`)

Reply:
send all, I think... if it's too noisy, then we can got for 5%